uwblueprint/childrens-aid-society
# Set up Chakra UI

## Notion ticket link

[Set up Chakra UI](https://www.notion.so/uwblueprintexecs/Setup-ChakraUI-c3233989c5284daaa3e3f8871c64eeb4)



## Implementation description
* added chakra UI + related dependencies to `package.json` by following [startup guide](https://v1.chakra-ui.com/guides/first-steps)
* set up barebones color/text styling files (component setup will come in another PR)
* changed our main pages to use chakra UI components



## Steps to test
1. Rebuild the container: `docker-compose down` && `docker-compose up --build`
2. Check that you're able to run the frontend locally without issues (you can navigate to `localhost:3000`)
3. Check that the pages load properly for you and look normal (`/`, `/intake`, `/visit`, `/login`)



## What should reviewers focus on?
* are the `theme/` files clear / easy to understand / look easy to add to?
* does it still run locally without issues?


## Checklist
